jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsExtension.java

```diff
@@ -44,14 +42,20 @@ public JaxrsExtension() {
     }
 
     enum JaxrsSubsystemModel implements SubsystemModel {
-        VERSION_4_0_0,
+        VERSION_4_0_0(ModelVersion.create(4, 0, 0)),
+        VERSION_5_0_0(ModelVersion.create(5, 0, 0)),
         ;
 
-        static final JaxrsSubsystemModel CURRENT = VERSION_4_0_0;
+        static final JaxrsSubsystemModel CURRENT = VERSION_5_0_0;
```

Comment:
I know we should bump this version to the next major one version from Brain's comment, but is there a reason we don't align this version with schema version ?

Reply:
They are just different things really. I don't recall how we ended up off. It had something to do with needing a model change because something was added. Either way, it's common for them not to match.

Reply:
Yes, IMHO we shouldn't attempt to align these. There are valid reasons for the numbers to get out of sync and that means it's not possible to keep them in alignment without doing things like skipping versions or introducing new models or schemas with no change. Both of which are confusing too.